sed: add German translation (#9434)
tldr-pages/tldr: pages.de/osx/sed.md

# sed

> Führe grundlegende Texttransformationen an der Eingabe aus.
> Weitere Informationen: <https://ss64.com/osx/sed.html>.

> - Ersetzt das erste Vorkommen einer Zeichenkette in einer Datei und gibt das Ergebnis aus:
> 
> `sed 's/{{finde}}/{{ersetze}}/' {{dateiname}}`

Reviewer: ```suggestion
- Ersetzt das erste Vorkommen einer Zeichenkette in einer Datei und gibt das Ergebnis aus:

`sed 's/{{finde}}/{{ersetze}}/' {{Pfad/zu/Datei}}`
```

Required fix: [broken placeholder](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths): `path/to/*` translated generic placeholder expected.

> `sed 's/{{finde}}/{{ersetze}}/' {{Pfad/zur/Datei}}`

Reviewer 2: ```suggestion
`sed 's/{{finde}}/{{ersetze}}/' {{pfad/zu/datei}}`
```

> - Ersetzt alle Vorkommen eines erweiterten regulären Ausdrucks in einer Datei:
> 
> `sed -E 's/{{regulärer_ausdruck}}/{{ersetze}}/g' {{dateiname}}`

Reviewer: ```suggestion
- Ersetzt alle Vorkommen eines erweiterten regulären Ausdrucks in einer Datei:

`sed -E 's/{{regulärer_ausdruck}}/{{ersetze}}/g' {{Pfad/zu/Datei}}`
```

The same issue as above.

> `sed -E 's/{{regulärer_ausdruck}}/{{ersetze}}/g' {{Pfad/zur/Datei}}`

Reviewer 2: ```suggestion
`sed -E 's/{{regulärer_ausdruck}}/{{ersetze}}/g' {{pfad/zu/datei}}`
```

> - Ersetzt alle Vorkommen einer Zeichenkette [i]n einer Datei, wobei die Datei überschrieben wird (d.h. an Ort und Stelle):
> 
> `sed -i '' 's/{{finde}}/{{ersetze}}/g' {{dateiname}}`

Reviewer: ```suggestion
- Ersetzt alle Vorkommen einer Zeichenkette [i]n einer Datei, wobei die Datei überschrieben wird (d.h. an Ort und Stelle):

`sed -i '' 's/{{finde}}/{{ersetze}}/g' {{Pfad/zu/Datei}}`
```

> `sed -i '' 's/{{finde}}/{{ersetze}}/g' {{Pfad/zur/Datei}}`

Reviewer 2: ```suggestion
`sed -i '' 's/{{finde}}/{{ersetze}}/g' {{pfad/zu/datei}}`
```

> - Ersetzt nur die Zeilen, die dem Zeilenmuster entsprechen:
> 
> `sed '/{{zeilenmuster}}/s/{{finde}}/{{resetze}}/' {{dateiname}}`

Reviewer: ```suggestion
- Ersetzt nur die Zeilen, die dem Zeilenmuster entsprechen:

`sed '/{{zeilenmuster}}/s/{{finde}}/{{resetze}}/' {{Pfad/zu/Datei}}`
```

The same issue as above.

> `sed '/{{zeilenmuster}}/s/{{finde}}/{{resetze}}/' {{Pfad/zur/Datei}}`

Reviewer 2: ```suggestion
`sed '/{{zeilenmuster}}/s/{{finde}}/{{resetze}}/' {{pfad/zu/datei}}`
```

> - Druckt nur den Text zwischen der n-ten Zeile bis zur nächsten Leerzeile:
> 
> `sed -n '{{zeilennummer}},/^$/p' {{dateiname}}`

Reviewer: ```suggestion
- Druckt nur den Text zwischen der n-ten Zeile bis zur nächsten Leerzeile:

`sed -n '{{zeilennummer}},/^$/p' {{Pfad/zu/Datei}}`
```

The same issue as above.

> `sed -n '{{zeilennummer}},/^$/p' {{Pfad/zu/Datei}}`

Reviewer 2: ```suggestion
`sed -n '{{zeilennummer}},/^$/p' {{pfad/zu/datei}}`
```

> - Mehrere Ausdrücke zum Suchen und Ersetzen auf eine Datei anwenden:
> 
> `sed -e 's/{{finde}}/{{ersetze}}/' -e 's/{{finde}}/{{ersetze}}/' {{dateiname}}`

Reviewer: ```suggestion
- Mehrere Ausdrücke zum Suchen und Ersetzen auf eine Datei anwenden:

`sed -e 's/{{finde}}/{{ersetze}}/' -e 's/{{finde}}/{{ersetze}}/' {{Pfad/zu/Datei}}`
```

The same issue as above.

> `sed -e 's/{{finde}}/{{ersetze}}/' -e 's/{{finde}}/{{ersetze}}/' {{Pfad/zur/Datei}}`

Reviewer 2: ```suggestion
`sed -e 's/{{finde}}/{{ersetze}}/' -e 's/{{finde}}/{{ersetze}}/' {{pfad/zu/datei}}`
```

> - Ersetzt das Trennzeichen `/` durch jedes andere Zeichen, das nicht in den Such- oder Ersetzungsmustern verwendet wird, z. B. `#`:
> 
> `sed 's#{{finde}}#{{ersetze}}#' {{dateiname}}`

Reviewer: ```suggestion
- Ersetzt das Trennzeichen `/` durch jedes andere Zeichen, das nicht in den Such- oder Ersetzungsmustern verwendet wird, z. B. `#`:

`sed 's#{{finde}}#{{ersetze}}#' {{Pfad/zu/Datei}}`
```

The same issue as above.

> `sed 's#{{finde}}#{{ersetze}}#' {{Pfad/zur/Datei}}`

Reviewer 2: ```suggestion
`sed 's#{{finde}}#{{ersetze}}#' {{pfad/zu/datei}}`
```

> # sed
> 
> > Führe grundlegende Texttransformationen an der Eingabe aus.
> > Weitere Informationen: <https://ss64.com/osx/sed.html>.
> 
> - Ersetzt das erste Vorkommen einer Zeichenkette in einer Datei und gibt das Ergebnis aus:
> 
> `sed 's/{{finde}}/{{ersetze}}/' {{dateiname}}`
> 
> - Ersetzt alle Vorkommen eines erweiterten regulären Ausdrucks in einer Datei:
> 
> `sed -E 's/{{regulärer_ausdruck}}/{{ersetze}}/g' {{dateiname}}`
> 
> - Ersetzt alle Vorkommen einer Zeichenkette [i]n einer Datei, wobei die Datei überschrieben wird (d.h. an Ort und Stelle):
> 
> `sed -i '' 's/{{finde}}/{{ersetze}}/g' {{dateiname}}`
> 
> - Ersetzt nur die Zeilen, die dem Zeilenmuster entsprechen:
> 
> `sed '/{{zeilenmuster}}/s/{{finde}}/{{resetze}}/' {{dateiname}}`
> 
> - Druckt nur den Text zwischen der n-ten Zeile bis zur nächsten Leerzeile:
> 
> `sed -n '{{zeilennummer}},/^$/p' {{dateiname}}`
> 
> - Mehrere Ausdrücke zum Suchen und Ersetzen auf eine Datei anwenden:
> 
> `sed -e 's/{{finde}}/{{ersetze}}/' -e 's/{{finde}}/{{ersetze}}/' {{dateiname}}`
> 
> - Ersetzt das Trennzeichen `/` durch jedes andere Zeichen, das nicht in den Such- oder Ersetzungsmustern verwendet wird, z. B. `#`:
> 
> `sed 's#{{finde}}#{{ersetze}}#' {{dateiname}}`
> 
> - Löscht die Zeile an der bestimmten Zeilennummer in einer Datei:
> 
> `sed -i '' '{{zeilennummer}}d' {{dateiname}}`

Reviewer: ```suggestion
# sed

> Text skriptfähig bearbeiten.
> Siehe auch: `awk`, `ed`.
> Weitere Informationen: <https://keith.github.io/xcode-man-pages/sed.1.html>.

- Ersetzen Sie alle Vorkommen von `apple` (Basic Regex) durch `mango` (Basic Regex) in allen Eingabezeilen und geben Sie das Ergebnis in `stdout` aus:

`{{Befehl}} | sed 's/Apfel/Mango/g''

- Führen Sie eine bestimmte Skript-[Datei] aus und geben Sie das Ergebnis in `stdout` aus:

`{{Befehl}} | sed -f {{Pfad/zu/script.sed}}`

- Ersetzen Sie alle Vorkommen von `apple` (erweiterte Regex) durch `APPLE` (erweiterte Regex) in allen Eingabezeilen und geben Sie das Ergebnis in `stdout` aus:

`{{Befehl}} | sed -E 's/(Apfel)/\U\1/g''

- Gib nur eine erste Zeile nach `stdout` aus:

`{{Befehl}} | sed -n '1p'`

- Ersetzen Sie alle Vorkommen von `apple` (einfacher Regex) durch `mango` (einfacher Regex) in einer `Datei` und speichern Sie eine Sicherungskopie des Originals in `file.bak`:

`sed -i bak 's/apple/mango/g' {{path/to/file}}`
```

Optional fix: this page can be synced with the [original](https://github.com/tldr-pages/tldr/blob/main/pages/osx/sed.md) one.

Reviewer 3: I don't understand what you're asking for here @EmilySeville7cfg?

Reviewer: I think I've asked to replace some translated placeholders to original ones where replacement occur. As this fix is optional and is broken (path placeholder is not translated) I can mark this conversation as resolved.

> - Löscht die Zeile an der bestimmten Zeilennummer in einer Datei:
> 
> `sed -i '' '{{zeilennummer}}d' {{dateiname}}`

Reviewer: ```suggestion
- Löscht die Zeile an der bestimmten Zeilennummer in einer Datei:

`sed -i '' '{{zeilennummer}}d' {{Pfad/zu/Datei}}`
```

The same issue as above.

> `sed -i '' '{{zeilennummer}}d' {{Pfad/zur/Datei}}`

Reviewer 2: ```suggestion
`sed -i '' '{{zeilennummer}}d' {{pfad/zu/datei}}`
```
